### PR TITLE
[F3D] Fix binary exports

### PR DESCRIPTION
**Pull request by Lilaa3**
If any flag was 0 this would fail

### CODE REVIEW

File: fast64_internal/f3d/f3d_gbi.py [hunk 1]
```diff
@@ -4236,7 +4236,10 @@ def geoFlagListToWord(flagList, f3d):
         if name in f3d.allGeomModeFlags:
             word += getattr(f3d, name)
         else:
-            raise PluginError("Invalid geometry mode flag " + name)
+            try:  # Try to cast name to an int instead, if this fails raise an explicit error
+                word += int(word, 0)
+            except:
+                raise PluginError("Invalid geometry mode flag " + name)
```
Comment (Dragorn421) [inline fast64_internal/f3d/f3d_gbi.py:4242]:
```suggestion
            except ValueError as e:
                raise PluginError("Invalid geometry mode flag " + name) from e
```
Catch-all except is bad practice, and chaining exceptions like this gives more helpful context in the trace

Comment (Lilaa3) [reply fast64_internal/f3d/f3d_gbi.py]: Fair, in this case the only exception that needs to be catched is value error, my bad

**Comment by Lilaa3** [comment]
[minimal example.blend.zip](https://github.com/user-attachments/files/16521900/minimal.example.blend.zip)
Since the user that reported this seems to be offline for now I made a minimal example, use the sm64 insertable binary exporter with ex2 as the microcode and a material with no geo mode flags enabled (there are other ways to trigger this but for the sake of simplicty this works), my file has this set up as is
I already tested and it fixed the issue!

**Review by Dragorn421 — APPROVED** [review]
fixes what it ought to fix

**Pull request merged**